infra/serverless/lambda-web-app-test/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "account_id" {
+  description = "Value of AWS Account"
+  type        = string
+  default     = "664450551143"

```

Comment:
Please remove. Make this anenvironment variable (I'd consider it something we don't want public and it maps to your AWS account when this should be something anyone with AWS can run (with their desired acct id set to an environment variables).

If you make the above changes please document in the README.md what type of environment variables A user needs setup as prereqs for running this code against their aws environment